packages/typescript/rules-snapshot.json

```diff
@@ -61,9 +116,27 @@
   "@typescript-eslint/prefer-as-const": "error",
   "@typescript-eslint/prefer-for-of": "error",
   "@typescript-eslint/prefer-function-type": "error",
+  "@typescript-eslint/prefer-includes": "error",
   "@typescript-eslint/prefer-namespace-keyword": "error",
+  "@typescript-eslint/prefer-nullish-coalescing": "error",
   "@typescript-eslint/prefer-optional-chain": "error",
+  "@typescript-eslint/prefer-readonly": "error",
+  "@typescript-eslint/prefer-reduce-type-parameter": "error",
+  "@typescript-eslint/prefer-string-starts-ends-with": "error",
+  "@typescript-eslint/promise-function-async": "error",
+  "@typescript-eslint/require-await": "error",
+  "@typescript-eslint/restrict-plus-operands": "error",
+  "@typescript-eslint/restrict-template-expressions": [
+    "error",
+    {
+      "allowAny": true,
```

Comment:
```suggestion
```
Not a blocker at all but as mentioned in https://github.com/MetaMask/eslint-config/pull/250#discussion_r1021645002 unless it causes a lot of noise there seems to be support for dropping `allowAny` as well and do the proper call where necessary.